docs/manual/common/concepts/InternalAndExternalCommunication.md

```diff
@@ -0,0 +1,7 @@
+#Internal and external communication
```

Comment:
It seems like this page is more about synchronous and asynchronous communication. It isn't clear to me at the moment what readers are meant to learn from this page.